.azure/publish-coverage.yml

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright(c) 2021 ADLINK Technology Limited and others
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License v. 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0, or the Eclipse Distribution License
+# v. 1.0 which is available at
+# http://www.eclipse.org/org/documents/edl-v10.php.
+#
+# SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+#
+
+#
+# Azure Pipeline specifically for building and submitting to Codecov
+#
+
+trigger: none
+pr: none
+
+schedules:
+  - cron: "0 12 * * 0"
+    branches:
+      include: [ master ]
+    always: false
+
+pool:
+  vmImage: ubuntu-20.04
+
+variables:
+- group: Default
+- name: python-version
+  value: 3.9
+
+steps:
+  - template: /.azure/templates/build-cyclone.yml
+  - bash: |
+      echo "###vso[task.setvariable variable=cyclonedds_home;]$(pwd)/cyclone-$(Agent.OS)"
+    name: set_cyclonedds_home
+    displayName: Set CYCLONEDDS_HOME
+  - template: /.azure/templates/build-test.yml
+  - bash: |
+      coverage combine src/cyclonedds/.coverage src/pycdr/.coverage
+      coverage xml
+    name: collect_test_coverage
+    displayName: Collect PyCDR and CycloneDDS coverage data
+  - bash:
+      bash <(curl -s https://codecov.io/bash) -t $(CODECOV_TOKEN) -f coverage.xml
```

Comment:
https://about.codecov.io/security-update/

Reply:
I will update this with the curl-uploader as used in the main cyclone repo

Reply:
Solved by using `codecov-python`, sadly depending on my own fork of it as the Azure Pipelines configuration was not present. I will submit a pull request and once that is merged and released we can change over to the proper package.
See here the report from a succesful run: https://codecov.io/gh/thijsmie/cyclonedds-python/commit/c2fdeafebbd5a098dc046e5ccbe811637a2887ce/